### ISSUE TITLE
[a11y] 11.1.1.1b Non-text content (graphics and objects)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1330]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

### Personal Space and Shared with me

- [x] Each directory or file in the list has "status" icons that show users whether the file has already been
users whether the file has already been shared, for example. Unfortunately
these icons are not displayed by screen readers, so this information is not accessible to
not accessible to screen reader users.
![image](https://github.com/owncloud/ios-app/assets/132444655/af2f94c7-a49a-491e-a3fd-eaaadf139478)

### Image Viewer

- [x] When focusing on the image with the screen reader, the description of the image is not displayed.
image, but information such as size and date entries.
![image](https://github.com/owncloud/ios-app/assets/132444655/64723d0b-66bc-4949-aed5-ab59b0a75c4e)
